### PR TITLE
Oracle Test Fix: #11554

### PR DESCRIPTION
**Pull request by ralmsdeveloper**
Oracle Test Fix: #11554

### CODE REVIEW

File: src/EFCore.Specification.Tests/TestUtilities/TestStore.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ protected TestStore(string name, bool shared)
             _shared = shared;
         }
 
-        public string Name { get; protected set; }
+        public virtual string Name { get; protected set; }
```
Comment (ralmsdeveloper) [inline src/EFCore.Specification.Tests/TestUtilities/TestStore.cs]: FYI: Some providers may need to rewrite the `Name` property, just like my provider and the Oracle example

@smitpatel 

cc/ @AndriySvyryd

**Comment by ralmsdeveloper** [comment]
@smitpatel 
Sorry, I sent a wrong command to my repository and closed it here.

It makes sense what you asked for, it got cleaner!

File: samples/OracleProvider/test/OracleProvider.FunctionalTests/TestUtilities/OracleTestStore.cs [hunk 2]
```diff
@@ -59,6 +59,8 @@ public static OracleTestStore CreateInitialized(string name)
             bool shared = true)
             : base(name, shared)
         {
+            Name = Name.Substring(0, Math.Min(Name.Length, 30));
```
Comment (smitpatel) [inline samples/OracleProvider/test/OracleProvider.FunctionalTests/TestUtilities/OracleTestStore.cs]: Just move it to call to `base`

Comment (ralmsdeveloper) [reply samples/OracleProvider/test/OracleProvider.FunctionalTests/TestUtilities/OracleTestStore.cs]: 🙈

**Comment by smitpatel** [comment]
@ralmsdeveloper - Thanks a lot of helping us out with maintaining Oracle provider sample running.

Merged via 5aa13c1ab5002abdd7a504e

**Pull request closed without merging**